src/SoftRobots.Inverse/binding/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ SP3_add_python_module(
     TARGET       ${PROJECT_NAME}
     PACKAGE      Bindings.Modules
     MODULE       SoftRobotsInverse
-    DESTINATION  Sofa
+    DESTINATION  /
```

Comment:
I prefer to import the module like this `Sofa.SoftRobotsInverse`.

Reply:
@adagolodjo can you check that the module is indeed in `build/lib/python3/site-packages/Sofa` when using `Sofa` as the `destination`? And could you try this when you have time: 
```console
python3 -c "import Sofa.SofaConstraintSolver; import Sofa.SoftRobotsInverse"
```